vendor-bin/doctrine_mongodb/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "autoload": {
```

Comment:
What `theofidry/composer-inheritance-plugin` does is actually merging a part of the config of the parent `composer.json` into this one.

For example if you check [this file](https://github.com/nelmio/alice/blob/master/vendor-bin/symfony/composer.json), you'll see that there is only that plugin and Symfony... because it inherits the whole autoloading and required packages config from the root one.

If you want more details on it, I suggest you to dig in https://github.com/theofidry/composer-inheritance-plugin, it's actually not much :)